tests/importable_test.py

```diff
@@ -1,26 +1,41 @@
 """Smoke tests related to loading entry points."""
 
+import typing
 from importlib.metadata import entry_points as _discover_entry_points
+from subprocess import check_call as _invoke_command
+from sys import executable as _current_runtime
 
 import pytest
 
 
+EXPECTED_ENTRY_POINTS: list[tuple[str, str, str]] = [
+    ('awx_plugins.credentials', 'aim', 'aim', 'aim_plugin'),
```

Comment:
The last components two are only used in the same place and concatenated always. No need to present them as disconnected inputs, because they are tightly coupled. Plus, taking into account that they are joined to the rest of the import path, I'd say the entire spec should be a param as a single unit.
```suggestion
    ('awx_plugins.credentials', 'aim', 'awx_plugins.credentials.aim:aim_plugin'),
```

Additionally, reusing the `'awx_plugins.credentials'` string this many times will :100: trip the WPS rule violation. It'd need to go into a constant.
Seeing that it's all the same in every param, I'd maybe move it into a separate `@pytest.mark.parametrize`, stacking two decorators together.
OTOH, if the observation of missing inventory plugin references is addressed, they'd need to have this structure. But still, this could be simplified a bit via something like `itertools.product()`.